tests/integration/modules/test_groupadd.py

```diff
@@ -14,10 +13,14 @@
 # Import Salt libs
 from salt.ext import six
 from salt.ext.six.moves import range
+import salt.utils
```

Comment:
The 2018.3 branch needs to use the new utils paths, so this should be `import salt.utils.platform` and the usage below needs to be adjusted as well. :)